pkg/resources/secret_common.go

```diff
@@ -51,62 +55,36 @@ var secretCommonSchema = map[string]*schema.Schema{
 	FullyQualifiedNameAttributeName: schemas.FullyQualifiedNameSchema,
 }
 
-type commonSecretCreate struct {
-	name     string
-	database string
-	schema   string
-	comment  *string
-}
-
-func handleSecretCreate(d *schema.ResourceData) commonSecretCreate {
-	create := commonSecretCreate{
-		name:     d.Get("name").(string),
-		database: d.Get("database").(string),
-		schema:   d.Get("schema").(string),
-	}
-	if v, ok := d.GetOk("comment"); ok {
-		create.comment = sdk.Pointer(v.(string))
+func handleSecretImport(d *schema.ResourceData) error {
+	if _, err := ImportName[sdk.SchemaObjectIdentifier](context.Background(), d, nil); err != nil {
+		return err
 	}
+	return nil
+}
 
-	return create
+func handleSecretCreate(d *schema.ResourceData) (database, schema, name string) {
```

Comment:
I think this function is not valuable. We do this for most of the resources without a special function for handling this. This can be inlined.

Reply:
will do